src/responder/nss/nss_protocol_grent.c

```diff
@@ -406,7 +406,6 @@ nss_protocol_fill_initgr(struct nss_ctx *nss_ctx,
         }
     }
 
-done:
     if (ret != EOK) {
         sss_packet_set_size(packet, 0);
```

Comment:
I think the final code would look more clean if you can move the 2 lines in the if-block to the error handling if-blokc before.